sentry/src/main/java/io/sentry/IScope.java

```diff
@@ -303,9 +304,14 @@ public interface IScope {
    *
    * @return the event processors list
    */
+  @ApiStatus.Internal
   @NotNull
   List<EventProcessor> getEventProcessors();
 
+  @ApiStatus.Internal
+  @NotNull
+  List<EventProcessorAndOrder> getOrderedEventProcessors();
```

Comment:
`l`: IIRC from our call we probably can remove one of those two due to not being used.

Reply:
I'll check again if it actually makes sense and follow up with another PR if we can remove it.